fieldservice_fsm_person_salary_extended/views/fsm_order_views.xml

```diff
@@ -1,20 +1,34 @@
 <?xml version="1.0" encoding="utf-8"?>
 <odoo>
 
-    <record id="view_fsm_order_request_workers_tree" model="ir.ui.view">
-        <field name="name">fsm.order.tree (FSM Order Request Workers)</field>
+    <record id="view_fsm_person_salary_extend_form" model="ir.ui.view">
+        <field name="name">fsm.order.form (FSM Person Salary Extended)</field>
         <field name="model">fsm.order</field>
         <field name="inherit_id" ref="fieldservice.fsm_order_form"/>
         <field name="arch" type="xml">
-            <xpath expr="//group//field[@name='person_ids']//tree" position="inside">
-                <field name="vehicle" />
-                <field name="payments_salary" />
-                <field name="payments_km" />
-                <field name="payments_diets" />
-                <field name="payments_advance" />
-                <field name="net_salary" />
+            <xpath expr="//group[@name='request_workers']" position="attributes">
+                <attribute name="invisible">1</attribute>
```

Comment:
In these cases, it is better to hide the field than the entire group, I understand that you have made a group with another name but I would keep it inside

Reply:
This is still missing, since the option of the new "group" with the name of salary lines is not quite correct